instrumental/driver_info.py

```diff
@@ -130,6 +141,11 @@
             'PM100D': ('Thorlabs', ['PM100D']),
         },
     }),
+    ('powermeters.thorlabs_tlpm', {
+        'params': ['name', 'reset'],
+        'classes': ['TLPM'],
+        'imports': [],
+    }),
```

Comment:
Ok I didn't know about this parse_modules  functionality (not seen in the doc, as far as I know). Could you just explained to me how the variable _INST_PARAMS and _INST_CLASSES are working together? Could not figure if I should place them in the module level (after imports) or in each classes
![image](https://user-images.githubusercontent.com/6609955/163125525-76fc9aaa-174b-4407-9a97-1cb0de3d4239.png)

Reply:
I just looked at the code (`instrumental/drivers/__init__.py`) to jog my memory. The "new" (seemingly undocumented) way is to define `_INST_PARAMS_` (note the _trailing_ underscore) as a class attribute. Then you can remove `_INST_PARAMS` and `_INST_CLASSES` from your module.

I think I mentioned this in another comment, but you can check out the Tektronix oscilloscope driver for an example of how this works.

Reply:
ok I got it, I modified the documentation according to this

Reply:
as for the _INST_PRIORITY priority one, I think it could also be related to one of my modifications, see 71371141b476c442d2317c7e3cd13868335c150b . If one choose one Paramset (with a particular classname in it) then one should be sure this is the one that is instantiated. That was not the case before 71371141b476c442d2317c7e3cd13868335c150b